server/views/components/goal-summary-card/_goalSummaryCardContent-goalInStatus-COMPLETED.njk

```diff
@@ -1,19 +1,30 @@
 <div class="govuk-summary-card__content">
   {% include './_goalSteps.njk' %}
 
-  {% if params.goal.notesByType.GOAL.length > 0 %}
-    {% set goalNote = params.goal.notesByType.GOAL[0] %}
-    <details class="govuk-details app-notes-expander" data-qa="overview-notes-expander">
-      <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
+  {% if params.goal.status === 'COMPLETED' %}
```

Comment:
I dont think we need this if statement (bearing in mind this is the template for a completed goal, so all goals that are rendered with this template will be completed 😁

Reply:
Removed.